src/main/java/com/conveyal/taui/controllers/BundleController.java

```diff
@@ -123,6 +112,9 @@ public static Bundle create (Request req, Response res) throws Exception {
                             Persistence.bundles.put(finalBundle.id, finalBundle);
                             return fs;
                         } catch (Exception e) {
+                            finalBundle.status = Bundle.Status.ERROR;
```

Comment:
Could you add a comment here on generally what kind of errors will be caught here?  For example, does this log or forward along GTFS validation errors?

Reply:
I'll add a comment. This is something that I plan to improve greatly in a future iteration.